backend/scripts/airtable/dumpProjects.js

```diff
@@ -46,23 +47,26 @@ async function dumpData() {
     .select({
       view: "All Proposals",
     })
-    .firstPage();
+    .all();
   if (data.length == 0) return console.error("Run this script again");
-  data.reverse();
+  data.reverse(); // reverse because old proposals might be missing some info
 
   const grantCategoryJsonReverse = {};
   Object.keys(grantCategoryJson).forEach((key) => {
     grantCategoryJsonReverse[grantCategoryJson[key]] = key;
   });
 
-  const done = [];
+  const allProjects = await Project.find({});
+
+  const done = allProjects.map((project) => project.projectName);
```

Comment:
Fairly self-explanatory.
Read previous version and this one, makes sense and looks good.
I have not tested it.